promising/configs.py

```
@@ -0,0 +1,178 @@
+        else:
+            # This is NOT a root PromiseConfig
+            self._inheritable_parent_config = self._parent_config.find_inheritable_config()
+            self._start_soon = get_concrete_value(start_soon, self._inheritable_parent_config.is_start_soon())
+            self._make_parent_wait = get_concrete_value(
+                make_parent_wait, self._inheritable_parent_config.is_make_parent_wait()
+            )
+            # TODO Split it into `config_inheritable` and `child_configs_inheritable` ?
+            #  (I don't remember what would be the use case for this, thought)
+            self._config_inheritable = get_concrete_value(config_inheritable, PromisingDefaults.CONFIGS_INHERITABLE)
+
+    def get_parent_config(self, raise_if_none: bool = True) -> Optional["PromiseConfig"]:
```

Ruff (0.14.14)

[error] Boolean-typed positional argument in function definition

(FBT001)